packages/quantic/force-app/main/default/lwc/quanticRecentResults/quanticRecentResults.js

```diff
@@ -0,0 +1,84 @@
+import {LightningElement, track, api} from 'lwc';
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+} from 'c/quanticHeadlessLoader';
+import {getItemfromLocalStorage, setIteminLocalStorage} from 'c/quanticUtils';
+
+import emptyListLabel from '@salesforce/label/c.quantic_EmptyRecentResultListLabel';
+
+export default class QuanticRecentResults extends LightningElement {
```

Comment:
I would suggest to rename the component `QuanticRecentResultList` to make it clearer that it is a list component.